packages/codemirror/src/editor.ts

```diff
@@ -166,7 +166,10 @@ export class CodeMirrorEditor implements CodeEditor.IEditor {
     // Turn off paste handling in codemirror since sometimes we want to
     // replace it with our own.
     editor.on('paste', (instance: CodeMirror.Editor, event: any) => {
-      if (!this._config['handlePaste']) {
+      if (
+        this._config['handlePaste'] !== undefined &&
+        !this._config['handlePaste']
+      ) {
```

Comment:
Slightly more succinct, if you want:

```suggestion
      const handlePaste = this._config['handlePaste'] ?? true;
      if (!handlePaste) {
```